### DISCUSSION TITLE
featured image pop up

### DISCUSSION THREAD

**Discussion by khoile2002** [hbstack/theme#2] [Ideas]
Very nice and fast theme. But i click on featured image. It pop up img. I think featured image Link to Post url better for reading behaviour.

Thank you!

**Comment by razonyang** [comment]
Make sense, will do a PR for this later.

**Comment by razonyang** [comment]
Done via https://github.com/hbstack/blog/pull/58, you'll need to upgrade the modules via

```sh
hugo mod get -u
```

or upgrade the specified modules if not working.

```sh
hugo mod get -u github.com/hbstack/blog
hugo mod get -u github.com/hbstack/bigger-picture
```

**Reply by lekhoi2002** [reply]
Many thanks!

**Reply by khoile2002** [reply]
Many thanks!

**Reply by razonyang** [reply]
@khoile2002  @lekhoi2002 you're welcome.
